src/together/resources/batch.py

```
@@ -72,6 +72,22 @@ def list_batches(self) -> List[BatchJob]:
         jobs = response.data or []
         return [BatchJob(**job) for job in jobs]
 
+    def cancel_batch(self, batch_job_id: str) -> BatchJob:
+        requestor = api_requestor.APIRequestor(
+            client=self._client,
+        )
+
+        response, _, _ = requestor.request(
+            options=TogetherRequest(
+                method="POST",
+                url=f"batches/{batch_job_id}/cancel",
+            ),
+            stream=False,
+        )
+
+        assert isinstance(response, TogetherResponse)
```

Let's not have an `assert` here. If we really want to check the response type, then perhaps better to throw an exception with a reasonable error message.